develop-docs/sdk/data-model/event-payloads/contexts.mdx

```diff
@@ -778,6 +778,42 @@ envelope endpoint.
 }
 ```
 
+## Page Context
+
+Page context contains information about the page that the event occurred on.
+
+`url`
+
+: **Required.** The URL of the page that the event occurred on.
+
+- Example: `https://sentry.io/issues?page=1`
+
+`user_agent`
+
+: _Optional_. The user agent of the page that the event occurred on.
+
+- Example: `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36`
```

Comment:
How can a "page" have a user agent?

Reply:
hmm, how do we phrase this? We need to send the user agent somehow, as the server infers the os/browser stuff from it 🤔 

We could make it `page_request`, would that make more sense?

Reply:
After thinking about this, I think it makes more sense to split this out into https://github.com/getsentry/sentry-docs/pull/13205